src/System.Private.CoreLib/src/System/Char8.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    /// <summary>
+    /// Represents a UTF-8 code unit, the elemental type of <see cref="Utf8String"/>.
+    /// </summary>
+    public readonly struct Char8 : IComparable<Char8>, IEquatable<Char8>
+    {
+        private readonly byte Value;
```

Comment:
Why isn't this named `_value` according to the [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)?

> 3. We use `_camelCase` for internal and private fields and use readonly where possible. Prefix internal and private instance fields with `_` [...]